test/annexB/language/literals/regexp/class-escape.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-regular-expressions-patterns
+es6id: B1.4
```

Comment:
This is a pattern already adopted through the tests, but in fact the section id is [`B.1.4`](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-regular-expressions-patterns).

For consistency it's good to follow the pattern on test262, omitting the first `.`, although, we could have a follow-up PR fixing it over the annexB tests.